src/test/java/christmas/MoneyTest.java

```diff
@@ -0,0 +1,27 @@
+package christmas;
+
+import christmas.domain.Money;
+import christmas.message.ExceptionMessage;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class MoneyTest {
+    @DisplayName("금액은 0원 이상인 경우 예외가 발생하지 않는다.")
+    @ParameterizedTest
+    @ValueSource(ints = {0, 10, 100, 1000, 999999, 1000000})
+    void moneySuccessTest(int amount) {
+        Assertions.assertThatNoException()
+                .isThrownBy(() -> new Money(amount));
+    }
+
+    @DisplayName("금액은 0원 미만인 경우 예외가 발생한다.")
+    @ParameterizedTest
```

Comment:
`@ParaneterizedTest`를 사용할때도 각각의 테스트 케이스 이름을 예쁘게 보여줄 수 있다는 사실, 알고 계셨나요?!
```suggestion
    @ParameterizedTest(name = "{0} 금액이 0원 미만이라면 예외가 발생한다.")
```

Reply:
와,, 이런 기능도 있었군요! 감사합니다! ㅎㅎ